src/tests/suite/dfs_unit_test.c

```diff
@@ -1906,6 +1908,13 @@ dfs_test_readdir(void **state)
 	assert_int_equal(rc, 0);
 }
 
+static void
+dfs_test_readdir(void **state)
+{
+	dfs_test_readdir_internal(state, OC_SX);
+	dfs_test_readdir_internal(state, OC_EC_2P2GX);
```

Comment:
we should add a check if this oclass is runnable on current system (i.e. should skip if we have less than 4 engines or whatever the rd level is for this oclass to work)

Reply:
ah, sure